mocks/oppgave-mock/src/main/java/no/nav/oppgave/OppgaveKontantstotteMockImpl.java

```diff
@@ -0,0 +1,96 @@
+package no.nav.oppgave;
+
+import static java.util.stream.Collectors.toList;
+
+import java.time.LocalDate;
+import java.util.List;
+
+import javax.validation.Valid;
+import javax.ws.rs.*;
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.UriInfo;
+import javax.ws.rs.container.ContainerRequestContext;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.swagger.annotations.*;
+
+
+@Produces(MediaType.APPLICATION_JSON)
+@Consumes(MediaType.APPLICATION_JSON)
+@Path("oppgave-kontantstotte/api/v1/oppgaver")
+public class OppgaveKontantstotteMockImpl {
+    private static final Logger LOG = LoggerFactory.getLogger(OppgaveKontantstotteMockImpl.class);
+
+    //TODO (TEAM FAMILIE) Lag mock-responser fra scenario NOSONAR
+    private Oppgave.Builder oppgaveMock = new Oppgave.Builder()
```

Comment:
Kan dette kobles mot modell, evt. REST-endepunkt for å sette ting i riktig state m.t.p. oppgaver? Slik at vi så langt det er mulig ikke benytter hardkodede verdier.

Reply:
Samme her.. kan prøve å gjøre klart oppsettet